app/views/search/_reverb_retirement_modal.html.erb

```diff
@@ -0,0 +1,17 @@
+<div class="modal fade sitetour" id="reverbRetirementModal" tabindex="-1" role="dialog">
+  <div class="modal-dialog" role="document">
+    <div class="modal-content">
+      <div class="modal-header">
+        <a href="#" class="close modal-close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></a>
+        <h2 class="modal-title" id="reverbRetirementModalLabel">Reverb is Retiring - Welcome to Earthdata Search</h2>
+      </div>
+      <div class="modal-body modal-body-welcome">
```

Comment:
After this `div` and before the first `p` in the modal content, we should place a heading that says "Welcome to Earthdata Search"